crates/bevy_internal/Cargo.toml

```diff
@@ -58,11 +58,14 @@ subpixel_glyph_atlas = ["bevy_text/subpixel_glyph_atlas"]
 webgl = ["bevy_core_pipeline?/webgl", "bevy_pbr?/webgl", "bevy_render?/webgl"]
 
 # enable systems that allow for automated testing on CI
-bevy_ci_testing = ["bevy_app/bevy_ci_testing", "bevy_render/ci_limits"]
+bevy_ci_testing = ["bevy_app/bevy_ci_testing", "bevy_render?/ci_limits"]
```

Comment:
What does this `?` do?

Reply:
Without the `?` enabling the `bevy_ci_testing` feature will unconditionally enable the optional `bevy_render` crate.
With it, `bevy_render/ci_limits` will only be enabled when `bevy_render` was already enabled.
It's unrelated to the rest of this PR.

Reply:
TIL. Great to know.